bot/chat-gpt/prompt.ts

```diff
@@ -41,7 +41,7 @@ export async function handlePrompt(client: APTClient, message: Message) {
         }
       ],
       temperature: 0.7,
-      model: 'gpt-3.5-turbo',
+      model: 'gpt-4',
```

Comment:
Should we use `gpt-4-1106-preview` instead ? So the price is x10 instead of x30 ? :D

Reply:
No thanks, `gpt-4` is already too beautiful for the audience we have haha

Reply:
Ah didn't check the actual prices haha let me have a look

Reply:
Non je pense que c'est bien cette version qu'on veut
https://openai.com/pricing